### PR TITLE
Update chrome extension to CDN v0.02

### PR DESCRIPTION
**Pull request by smalls**

### CODE REVIEW

File: extension/new-tab.js [hunk 1]
```diff
@@ -5,11 +5,18 @@
...
+// TODO(smalls) - there should be a better system of unique ids
+var faux_gid = 2000;
+
 /**
  * Load the current browsing data from all non-https tabs on all devices.
  */
 async function loadBrowsingData(arc, manifest) {
 
+  let entityKlass = manifest.findSchemaByName('Tab').entityClass();
+  let view = arc.createView(entityKlass.type.viewOf(), 'tabView'); // tabView could be 'anon'
```
Comment (dstockwell) [inline extension/new-tab.js]: OK to start, but I think the views that contain the browsing data should be constructed in a manifest instead `... new Manifest(); manifest.newView(...)`. They would then get mapped/copied into the arc as necessary by the suggested recipe.

File: extension/new-tab.js [hunk 2]
```diff
@@ -5,11 +5,18 @@
 // subject to an additional IP rights grant found at
 // http://polymer.github.io/PATENTS.txt
 
+
+// TODO(smalls) - there should be a better system of unique ids
+var faux_gid = 2000;
+
 /**
  * Load the current browsing data from all non-https tabs on all devices.
```
Comment (dstockwell) [inline extension/new-tab.js]: what is the non-https restriction here?

File: extension/new-tab.js (hunk 1 above)
Comment (dstockwell) [reply extension/new-tab.js]: Although that's probably 0.03 or 0.0.4 :P

File: extension/new-tab.js [hunk 3]
```diff
@@ -5,11 +5,18 @@
...
  */
 async function loadBrowsingData(arc, manifest) {
 
+  let entityKlass = manifest.findSchemaByName('Tab').entityClass();
```
Comment (dstockwell) [inline extension/new-tab.js]: Everything is a `Tab` for now?

File: extension/new-tab.js (hunk 2 above)
Comment (smalls) [reply extension/new-tab.js]: Chalk this up to me mis-understanding a regex below. I'll remove this in the future (just a comment saying 'loads browsing data' isn't particularly helpful).

File: extension/new-tab.js (hunk 3 above)
Comment (smalls) [reply extension/new-tab.js]: Yes, but this was a mistake. I'll send a review later today with a fixed version.

File: extension/new-tab.js (hunk 1 above)
Comment (smalls) [reply extension/new-tab.js]: Oh good to know! I was going to ask about when to prefer one or the other - it's an interesting question, though, so I'll ask in the chat so we can all discuss.

**Pull request merged**